app/assets/javascripts/hyrax/fileupload.js

```diff
@@ -0,0 +1,9 @@
+//= require hyrax/uploader
+// This file is the default initialization of the fileupload.  If you want to call
+// hyraxUploader with other options (like afterSubmit), then override this file.
+Blacklight.onLoad(function() {
+  var options = {};
+  $('#fileupload').hyraxUploader(options);
+  $('#fileuploadlogo').hyraxUploader({downloadTemplateId: 'logo-template-download'});
+  $('#fileuploadthumbnail').hyraxUploader({downloadTemplateId: 'thumbnail-template-download'});
```

Comment:
were logo and banner upload broken before? if so this could use an override comment

Reply:
logo and banner were working prior